packages/app/src/__deprecated__/renderer/components/core/tooltip/element-with-tooltip.component.tsx

```diff
@@ -101,6 +105,26 @@ const ElementWithTooltip: FunctionComponent<Props> = ({
     }
   }
 
+  useLayoutEffect(() => {
+    const trigger = () => {
+      if (showIfTextEllipsis) {
+        const text = ref.current as HTMLDivElement
+        const isEllipsisActive = text && text.offsetWidth < text.scrollWidth
+        setIsEllipsis(isEllipsisActive ? isEllipsisActive : false)
+      }
+    }
+
+    if (ref.current) {
+      if ("ResizeObserver" in window) {
+        new ResizeObserver(trigger).observe(ref.current)
```

Comment:
It would be good to check, but probably we should `disconnect` or `unobserve` on `ResizeObserver` when destroying the component (https://stackoverflow.com/questions/67581868/should-i-call-resizeobserver-unobserve-for-removed-elements)

Reply:
Changed in https://github.com/mudita/mudita-center/pull/1030/commits/f983a62c20cec40ef8e1441cc205e51683274ef5